### DISCUSSION TITLE
modal is not rendering in the intercept + parallel route combination

### DISCUSSION THREAD

**Discussion by aryan877** [vercel/next.js#63638] [App Router]
is my setup correct? athlete/jobs/job is not rendering the modal.... I'm not being directed to the page... but the modal is not showing up... it means my route is being intercepted but nothing renders 

i have copied the nextgram example 

I have added the modal prop in the layout file as well

<img width="339" alt="Screenshot 2024-03-23 at 4 40 23 PM" src="https://github.com/vercel/next.js/assets/38075578/72943b9a-75ec-46ae-8052-dd2918003a7a">

**Comment by Moganesan** [comment]
this is my setup the modal from app/payments/links/@modal/(.)create/page.tsx is opening but its not rendering the root layout. but i have one more modal in my project that is app/products/@modal/(.)create.tsx that works expected opening on top of the root layout.

let me know if you have a solution for it.

for your issue i think you have to remove unwanted folder like (app) inside app directory and make sure you are following the doc.

<img width="305" alt="Screenshot 2024-10-08 at 9 06 11 PM" src="https://github.com/user-attachments/assets/e1caeed1-04f3-4660-9a61-cae5c0d8d586">
